backend/compat.py

```diff
@@ -1 +1,13 @@
+# Werk van Sil
 
+# for-loop
+# [] 
+
+
+def validate_appointment(data: dict):
+    KeyList = ['title', 'description', ...]
+    for key in KeyList:
+        if key not in dict:
```

Comment:
Hier staat trouwens `key not in dict`. De syntax `def validate_appointment(data: dict)` betekent dat de naam/variabele `data` `type` `dict` zou moeten zijn. De variabele naam is dus `data` dus je moet `dict` even vervangen door `data`